docs/api/browser-window.md

```diff
@@ -1236,7 +1236,11 @@ Captures a snapshot of the page within `rect`. Omitting `rect` will capture the
   * `postData` ([UploadRawData[]](structures/upload-raw-data.md) | [UploadFile[]](structures/upload-file.md) | [UploadBlob[]](structures/upload-blob.md)) (optional)
   * `baseURLForDataURL` String (optional) - Base url (with trailing path separator) for files to be loaded by the data url. This is needed only if the specified `url` is a data url and needs to load other files.
 
-Same as `webContents.loadURL(url[, options])`.
+Returns `Promise` - the promise will resolve when the page has finished loading
```

Comment:
Can we be super specific what the `Promise` type is here; if you don't declare the type it defaults to `any`.  I believe here this should be `Promise<void>` indicating the promise resolves with nothing.

(Check the archeologist for the different in types generation)

(Same for all other new promise docs in this PR)

Reply:
done